### DISCUSSION TITLE
How to change display brightness more than 50% by ddc

### DISCUSSION THREAD

**Discussion by MKingWang** [MonitorControl/MonitorControl#864] [General]
hI 
my desplay is aoc LV273HUPR，if i check `Use hardware DDC control`, the brightness is no change more than 50%
macos: 12.1

**Comment by waydabber** [comment]
Hi, your display might be incompatible or there is an other issue. Since there is no info on your setup, it is hard to know. You can try alternate apps like ddctl or m1ddc and see if they can control the display.

**Comment by waydabber** [comment]
(note that the first 50% of brightness setting is done by software and not using DDC with the default settings so DDC probably does not work).

**Reply by MKingWang** [reply]
thanks! so i cant use ddc and must uncheck Use hardware DDC control to make brightness 50%-100% working?

**Reply by waydabber** [reply]
Hi, yes, if you uncheck that option, then software (gamma) control will take care of 0-100% for that display. DDC control is usually superior however so it might be worth it to try getting to the bottom of the issue.

**Reply by MKingWang** [reply]
Thanks a lot
